inspirehep/modules/records/utils.py

```diff
@@ -24,6 +24,9 @@
 
 from __future__ import absolute_import, division, print_function
 
+import requests
+from contextlib import closing, contextmanager
+from urlparse import urlparse
```

Comment:
`urlparse` should come from `six` for future Python 3 compatibility. See: https://pythonhosted.org/six/#module-six.moves